hardware_interface/include/hardware_interface/helpers.hpp

```diff
@@ -0,0 +1,47 @@
+// Copyright 2023 ros2_control Development Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef HARDWARE_INTERFACE__HELPERS_HPP_
+#define HARDWARE_INTERFACE__HELPERS_HPP_
+
+#include <locale>
+#include <sstream>
+#include <stdexcept>
+#include <string>
+
+namespace hardware_interface
+{
+
+/** \brief Helper function to convert a std::string to double in a locale-independent way.
+ \throws std::runtime_exception if not a valid number
```

Comment:
We're throwing `std::invalid_argument`, so let's update this comment?

Reply:
Oh, I just copied that over and didn't pay any attention to it. Thx, I fixed it!